### ISSUE TITLE
interface error

### ISSUE THREAD

**Issue by turtle0617** [taozhi8833998/node-sql-parser#122]
thanks for your great work. but I find a interface error.
hope you can read my issue and fix that.

input:
```
const { Parser } = require("node-sql-parser");
const parser = new Parser();
const grammar = "update userinfo set userpwd='111',email='[email protected]';\n 
delete from userinfo where username='yyy';";
const ast = parser.astify(grammar, opt);
```
output:
```
[ { type: 'update',
    table: [ { db: null, table: 'userinfo', as: null } ],
    set:
     [ { column: 'userpwd',
         value: { type: 'string', value: '111' },
         table: null },
       { column: 'email',
         value: { type: 'string', value: '[email protected]' },
         table: null } ],
    where: null } ,
{ type: 'delete',
    table:
     [ { db: null, table: 'userinfo', as: null, addition: true } ],
    from: [ { db: null, table: 'userinfo', as: null } ],
    where:
     { type: 'binary_expr',
       operator: '=',
       left: { type: 'column_ref', table: null, column: 'username' },
       right: { type: 'string', value: 'yyy' } } }]
```
should be `from: Array<From | Dual> | null;`
https://github.com/taozhi8833998/node-sql-parser/blob/054d6042a5a4e689385f73de58074a69fbebb279/index.d.ts#L75

 should be `type: 'update';`
https://github.com/taozhi8833998/node-sql-parser/blob/054d6042a5a4e689385f73de58074a69fbebb279/index.d.ts#L73

should be ` table: any;`
https://github.com/taozhi8833998/node-sql-parser/blob/054d6042a5a4e689385f73de58074a69fbebb279/index.d.ts#L81

**Comment by taozhi8833998** [comment]
thanks for your review, I have fixed it.

**Comment by Val-istar-Guo** [comment]
https://github.com/taozhi8833998/node-sql-parser/blob/792cf2da5a59bef119a255baf2b317e2c290becf/index.d.ts#L68

Insert have the same issue @taozhi8833998

**Comment by taozhi8833998** [comment]
> https://github.com/taozhi8833998/node-sql-parser/blob/792cf2da5a59bef119a255baf2b317e2c290becf/index.d.ts#L68
> 
> Insert have the same issue @taozhi8833998

fix in `1.6.18`
